pkg/autoscaler/config/config.go

```diff
@@ -191,6 +197,10 @@ func validate(lc *autoscalerconfig.Config) (*autoscalerconfig.Config, error) {
 		return nil, fmt.Errorf("max-scale-limit = %v, must be at least 0", lc.MaxScaleLimit)
 	}
 
+	if lc.MinScale > lc.MaxScale {
```

Comment:
what if lc.MaxScale hasn't been set, so is == 0?

Reply:
added validation and test